### ISSUE TITLE
Add option to Create Multicam Clips

### ISSUE THREAD

**Issue by TheDaveChap** [latenitefilms/BRAWToolbox#18]
I would love an option to bring the BRAW clips in as Compound Clips with the audio instead of Synced clips. This would allow for changes to ripple through to the edit/timeline if a change needed to be made to audio patching, processing and color tweaks by opening up the CC and making the change. The way Synced clips work, you can't make a global change to a clip inside that Synced clip timeline—they are all individual copies unless you happen to make this change in the browser before editing the clip. Otherwise you'd have to do a ton of match and replace all your edits with the updated clip in the browser.

**Comment by latenitefilms** [comment]
This is a slightly tricky one. The main reason for doing sync-clips is so that if you're using dual-system audio, you can use something like Sync-N-Link to sync the sync-clips with your location audio and create new multi-cam clips.

I could add the option to automatically create a new multi-cam clip that contains each sync-clip?

I didn't want to create multi-cam clips to start with, as it seems a bit weird to embed a multi-cam within a multi-cam if using dual-system audio.

Using compound clips instead of sync-clip is an option, but I think it just complicates things in terms of exporting FCPXMLs to Resolve, etc.

Generally speaking, if I'm not using dual-system audio, I always create new multi-cam clips anyway, so I have that control.

I did think of using "normal clips" instead of a sync-clip, but then you can't easily double click on them to control the metadata.

**Comment by latenitefilms** [comment]
Thinking about this some more, as mentioned above, I think one potential compromise is probably to create "normal" video clips for the main BRAW file, then put that clip inside a multi-cam.

<img width="639" alt="image" src="https://user-images.githubusercontent.com/22286696/210979077-28adeab2-47b1-4c2b-ad6d-bb9e0f14c337.png">

The downside is that this makes it harder to get to the BRAW metadata, as you need to go inside the multi-cam clip, then select the video clip and select "Clip > Open Clip". However, I can't imagine people will be messing with the RAW metadata very often. Ideally you basically set everything you want in the Workflow Extension.

An easier fix would be to just leave it as a sync-clip, but wrap it in a Multicam clip, so you can still double click the angle to get to the RAW controls. If you're using dual-system audio, you'd delete these multi-cam clips and create new ones with the sync-clips.

I've also looked into just replacing sync-clips with compound clips, and it would be a fair bit of re-engineering to swap them over, but certainly possible. The downside is that I feel like FCPX might struggle on a big project if it's filled with Compound Clips. I also feel like it might be harder to conform in Resolve for colour grading?

Currently none of these options work with Sync-N-Link X, and I've reached out to Gregory. I don't think they ever envisaged getting a FCPXML with no actual video files inside. Hopefully it's a relatively easy fix.

Long story short... I'm still not sure what the way solution is.

**Comment by latenitefilms** [comment]
I'm currently leaning on the idea of just creating a multi-cam clip instead of a sync-clip. This probably gives the most flexibility.

**Comment by TheDaveChap** [comment]
Maybe synced clips are the way to go then. I was thinking if they were compound clips, it would be easier to open and adjust the Braw controls (white balance for example) during the edit, where the synced clips are all silos once you begin editing. Thinking of broll shoots. But for an interview, i can make a Multicam and I guess if I open that synced clip timeline from within the Multicam, those Braw changes will come back through… I just can’t open the synced clip and make changes once the Multicam is made, which is fine since I tend to reject the clips ones the Multicam is made.

Multicams and Compound clips render in the background, so having them as synced clips will avoid rendering for people who have that turned on by default too.

**Comment by latenitefilms** [comment]
Yes, I think you're right. I'll leave as Synchronised Clips for now, as I think that makes the most sense.

However, I'll have a think about adding an option to "make multi-cams of sync-clips" - so that BRAW Toolbox can automatically create multi-cam clips for all your B-Roll footage, so you don't have to make multi-cams or compound clips manually.

**Comment by latenitefilms** [comment]
The next beta will have a new settings button with the option to "Create Multicam Clips":

<img width="230" alt="image" src="https://user-images.githubusercontent.com/22286696/211122348-3ca88b23-9148-4f3b-89e9-7f4082341d33.png">
